audioipc/src/rpccore.rs

```diff
@@ -118,24 +115,23 @@ impl<Request, Response> Proxy<Request, Response> {
 
 impl<Request, Response> Clone for Proxy<Request, Response> {
     fn clone(&self) -> Self {
```

Comment:
Is `clone` the reason why `key` needs to be a `Result` since `clone` implementation (for `Clone` trait) needs to return a `Self`? If so, have you considered using a custom `clone` function to get away from this limitation?

Reply:
Yeah, that was the reason.  I've switched to `try_clone` instead, that's a cleaner solution.